src/main/java/com/metamx/cache/CaffeineCacheConfig.java

```diff
@@ -29,6 +30,11 @@
   @JsonProperty
   private long maxSize = -1;
 
+  @JsonProperty
+  // Do not use DEFAULT unless you're running 8u60 or higher
+  // see https://github.com/ben-manes/caffeine/issues/77
+  private CacheExecutor cacheExecutor = CacheExecutor.SINGLE_THREAD;
```

Comment:
because `DEFAULT` is broken in many java installations. I could rename it to COMMON_FJP if that makes it clearer